data/tlg5034/tlg001a/tlg5034.tlg001a.perseus-grc1.xml

```diff
@@ -48,20 +48,23 @@
                         <monogr>
                             <author>Scholia Pindar</author>
                             <title>Scholia in Pindarum Olympian Odes</title>
```

Comment:
Also add an `xml:lang="mul"` attribute to this `<title>` element since the text of this element is made up of multiple languages.